MaterialLib/Fluid/Viscosity/createViscosityModel.cpp

```diff
@@ -32,7 +32,7 @@ namespace Fluid
 static std::unique_ptr<FluidProperty> createLinearPressureDependentViscosity(
```

Comment:
The first character of this file name and that in createDensityModel.cpp will be changed to its capital one, "C".